### PR TITLE
Target 6/28 -  NDK v1.1.0 & Android agent v7.5.0

### PR DESCRIPTION
**Pull request by ywang-nr**
* What problems does this PR solve? - Release notes for NDK v1.1.0 & Android agent v7.5.0, log forwarding docs

### CODE REVIEW

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-ndk-110.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+subject: Android agent
+title: Android NDK agent v1.1.0
+releaseDate: '2024-06-27'
+version: 1.1.0
+---
+
+## New
+
+* Reduce the overall size of the agent
+* Add feature flag to have the ability to disable ANR detecture in the agent
```
Comment (ywang-nr) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-ndk-110.mdx:11]: detection

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-750.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+subject: Android agent
+releaseDate: '2024-06-28'
+version: 7.5.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.5.0.zip'
+---
+
+## New
+
+* Add mobile log reporting feature. You can read our docs to learn more and how to [enable this feature](/docs/mobile-monitoring/mobile-monitoring-ui/logs).
```
Comment (rhetoric101) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-750.mdx:10]:
```suggestion
* Add mobile log reporting feature. You can read our docs to learn more and how to [enable this feature](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-logs/).
```

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-ndk-110.mdx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+subject: Android agent
+title: Android NDK agent v1.1.0
+releaseDate: '2024-06-27'
+version: 1.1.0
+---
+
+## New
+
+* Reduce the overall size of the agent.
+* Extend AgentNDK.Builder.withANRMonitor(boolean) to disable native ANR detection as well.
```
Comment (rhetoric101) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-ndk-110.mdx:11]: @ywang-nr is there a way for us mark this up so it is more readable? For example, could we put it inside back ticks or something: `gentNDK.Builder.withANRMonitor(boolean)`

Is the (boolean) part of the expression?

**Review by rhetoric101 — COMMENTED** [review]
@ywang-nr thanks for adding these release notes. Could you take a look at my comments?

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-ndk-110.mdx (hunk 3 above)
Comment (ywang-nr) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-ndk-110.mdx:11]: Yes, (boolean is part of the expression), the suggested changes are good.

**Review by rhetoric101 — APPROVED** [review]
@ywang-nr thanks for making this last-minute improvements!

**Pull request merged**